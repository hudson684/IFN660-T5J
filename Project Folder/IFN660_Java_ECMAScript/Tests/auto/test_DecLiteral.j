0
123
1_2
12391
999
9
12_34_56