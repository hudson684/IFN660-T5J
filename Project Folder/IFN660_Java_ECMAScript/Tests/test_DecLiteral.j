valid
'a'
'%'
'\t'
'\\'
'\''
'\u03a9'
'\uFFFF'
'\177'
'™'

invalid
_52
52_
__52
52__
5_2_