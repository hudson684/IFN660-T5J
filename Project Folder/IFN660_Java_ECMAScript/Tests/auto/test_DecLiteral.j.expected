IntegerLiteral (0)
IntegerLiteral (123)
IntegerLiteral (12)
IntegerLiteral (12391)
IntegerLiteral (999)
IntegerLiteral (9)
IntegerLiteral (123456)
EOF